web/static/js/reducers/questionnaire.js

```diff
@@ -150,28 +166,28 @@ type ActionChangeStepSmsPrompt = {
   newPrompt: string
 };
 
-const changeStepSmsPrompt = (state, action: ActionChangeStepSmsPrompt) => {
+const changeStepSmsPrompt = (state, action: ActionChangeStepSmsPrompt, quiz: Questionnaire): Step[] => {
   return changeStep(state, action.stepId, step => ({
     ...step,
     prompt: {
       ...step.prompt,
-      'en': {
-        ...step.prompt['en'],
+      [quiz.defaultLanguage]: {
```

Comment:
Another place to check: here I changed the assumption of editing English to the slightly less static assumption that we're editing the default language. Neither is actually right. I guess we should augment all relevant actions with a "lang" prop to use here.